patches/nginx-1.13.6-socket_cloexec.patch

```diff
@@ -0,0 +1,127 @@
+diff --git a/auto/unix b/auto/unix
+index 10835f6c..d5d5de6a 100644
+--- a/auto/unix
++++ b/auto/unix
+@@ -990,3 +990,16 @@ ngx_feature_test='struct addrinfo *res;
+                   if (getaddrinfo("localhost", NULL, NULL, &res) != 0) return 1;
+                   freeaddrinfo(res)'
+ . auto/feature
++
++ngx_feature="SOCK_CLOEXEC support"
++ngx_feature_name="NGX_HAVE_SOCKET_CLOEXEC"
++ngx_feature_run=no
++ngx_feature_incs="#include <sys/types.h>
++                  #include <sys/socket.h>
++                  #include <fcntl.h>"
++ngx_feature_path=
++ngx_feature_libs=
++ngx_feature_test="int fd;
++                  fd = socket(AF_INET, SOCK_STREAM | SOCK_CLOEXEC, 0);
++                  fcntl(fd, F_SETFD, FD_CLOEXEC);"
```

Comment:
Seems like you misunderstood me :)

It's better to make these 2 separate feature tests. They are different things. So that maybe we can remove the win32 special handling and it will also automatically work on other systems lacking support for even the `FD_CLOEXEC` thing. It's safer and more generic.